packages/migration-graph/tests/fixtures/project.ts

```diff
@@ -0,0 +1,109 @@
+import { Project } from 'fixturify-project';
+import { dirname, join } from 'path';
+import merge from 'lodash.merge';
+import tmp from 'tmp';
+import { execSync } from 'child_process';
+import { existsSync } from 'fs-extra';
+
+tmp.setGracefulCleanup();
+
+import {
+  getEmberAppFiles,
+  getEmberAppWithInRepoAddonFiles,
+  getEmberAddonFiles,
+  getEmberAppWithInRepoEngine,
+} from './files';
+import rimraf from 'rimraf';
+
+// this scenario represents the last Ember 3.x release
+export async function ember3(_project: Project) {}
+
+export function getEmberAppProject(project: Project = emberAppTemplate()): Project {
```

Comment:
I think this is ok for now. But for ease of API and redundancy maybe all the variants of getEmber* can be in the same function with a switch statement around type. eg.

```
export function getEmberFixture(name: string, projectType: 'addon' | 'engine' | 'app') {
  switch (projectType) {}
  // ...
}
```

Reply:
Made the changes.

At first, I used an enum, then I realized that a type is simpler and just as good. No need for the overhead of an enum. Thanks for the suggestion.